src/util/options.schema.json

```diff
@@ -1737,6 +1837,44 @@
               "type": "boolean",
               "default": true
             },
+            "narrow-reuse": {
+              "title": "solvers.td3.narrow-reuse",
+              "description": "Reuse value when switching from widening to narrowing phase. Avoids one unnecessary re-evaluation.",
+              "type": "boolean",
+              "default": true
+            },
+            "narrow-reuse-verify": {
+              "title": "solvers.td3.narrow-reuse-verify",
+              "description": "Instead of reusing value when switching from widening to narrowing phase, just verify that the reuse would be correct by re-evaluating.",
+              "type": "boolean",
+              "default": false
+            },
```

Comment:
Should we keep this option?

Reply:
>Do you mean just solvers.td3.narrow-reuse-verify because https://github.com/goblint/analyzer/pull/443 is already in master?

I was actually not aware of this option; but yes, if the option `solvers.td3.narrow-reuse-verify` is no longer needed (without the abort), the code would be a bit cleaner without it.